### ISSUE TITLE
TypeError: a bytes-like object is required, not 'str' when running with python3

### ISSUE THREAD

**Issue by zskdan** [kaitai-io/kaitai_fs#18]
When I run the rar fs under python3.6 I got the following exception:
```
 $ python3 -m kaitaifs.cli rar examples/test.rar  mount
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/zakaria/workspace/kaitai_fs/kaitaifs/cli.py", line 58, in <module>
    main()
  File "/home/zakaria/workspace/kaitai_fs/kaitaifs/cli.py", line 51, in main
    fs_class(args.image_file),
  File "/home/zakaria/workspace/kaitai_fs/kaitaifs/fs/rar_fs.py", line 10, in __init__
    super(RarFS, self).__init__()
  File "/home/zakaria/workspace/kaitai_fs/kaitaifs/kaitai_simple_fs.py", line 42, in __init__
    self.generate_tree()
  File "/home/zakaria/workspace/kaitai_fs/kaitaifs/fs/rar_fs.py", line 15, in generate_tree
    self.add_obj_to_path(block.body.file_name.split('\\'), block)
TypeError: a bytes-like object is required, not 'str'
```
No issue is noticed when with python2

**Comment by zskdan** [comment]
It seems that https://github.com/kaitai-io/kaitai_fs/pull/19 fix the issue.

**Comment by generalmimon** [comment]
Should be resolved with #19 then, so closing.
